crates/kitsune_p2p/dht/.gitignore

```diff
@@ -0,0 +1 @@
+Cargo.lock
```

Comment:
delete this gitignore - the lock file won't be generated in this directory anyways...